cms/djangoapps/contentstore/views/videos.py

```diff
@@ -134,9 +134,9 @@ def videos_handler(request, course_key_string, edx_video_id=None):
 
     if request.method == "GET":
         if "application/json" in request.META.get("HTTP_ACCEPT", ""):
-            return videos_index_json(course)
+            return videos_index_json(course, request)
         else:
-            return videos_index_html(course)
+            return videos_index_html(course, request)
```

Comment:
keeping it consistent with existing code e.g. videos_post method uses course, request